sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1259,15 +1261,15 @@ def CreateItem(
         document: Dict[str, Any],
         options: Optional[Mapping[str, Any]] = None,
         **kwargs: Any
-    ) -> Dict[str, Any]:
+    ) -> CosmosDictResponse:
```

Comment:
```suggestion
    ) -> CosmosDictResponse[str, Any]:
```

Reply:
pylint is complaining about this for every single change of it - probably since it is an object at this point? I'm thinking we could just rename it to `CosmosDict` if that makes it more obvious that it is the same dictionary, but I'm not sure that fully addresses this lack of clarity from the new rtype

Reply:
Aaaah I _think_ the reason is that we're running the pylint and mypy passes with Python 3.8, whereas subtyping for `dict` and `list` is supported from 3.9. I think you should be able to replace the inherited classes with typing.Dict and typing.List to make it work.
```python
class CosmosDictResponse(Dict):
    ...
```

Reply:
As an aside - I'm also not against using the shorter `CosmosDict` and `CosmosList`....

Reply:
pylint still did not allow for any subtyping so I removed it